modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -40,7 +40,8 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>{{#springFoxDocumentationProvider}}2.5.8{{/springFoxDocumentationProvider}}{{^springFoxDocumentationProvider}}2.6.3{{/springFoxDocumentationProvider}}</version>
+        <version>{{#springFoxDocumentationProvider}}2.5.10{{/springFoxDocumentationProvider}}{{^springFoxDocumentationProvider}}2.6.4{{/springFoxDocumentationProvider}}</version>
+        <relativePath/> <!-- lookup parent from repository -->
```

Comment:
Is this required or just recommended to add?

Reply:
It's the way [start.spring.io](https://start.spring.io) recommends. It is required when you have a multi-module maven project where the generated model is a sub module.